app/templates/search.html

```diff
@@ -7,6 +7,56 @@
 
 {% block content %}
 {% if pages_required > -1 %}
+  <table class="filter">
+    <tr>
+      <th>Sort By:</th>
+      {% if sort == 'mmr' %}
+        <td class="isActive">MMR</td>
+      {% else %}
+        <td><a href="?region={{ region }}&rank={{ rank_filter }}&sort=mmr&page=1">MMR</a></td>
+      {% endif %}
```

Comment:
Instead of 
```     
{% if sort == 'mmr' %}
  <td class="isActive">MMR</td>
{% else %}
  <td><a href="?region={{ region }}&rank={{ rank_filter }}&sort=mmr&page=1">MMR</a></td>
{% endif %}
```

could we do:
```      
<td {% if sort == 'mmr' %}class="isActive"{% endif %}><a href="?region={{ region }}&rank={{ rank_filter }}&sort=mmr&page=1">MMR</a></td>
```

and pretty much everywhere else with this format?